ext/opentelemetry-ext-opentracing-shim/tests/testbed/test_listener_per_request/README.rst

```diff
@@ -0,0 +1,19 @@
+
+Listener Response example.
+==========================
+
+This example shows a ``Span`` created upon a message being sent to a ``Client``\ , and its handling along a related, **not shared** ``ResponseListener`` object with a ``on_response(self, response)`` method to finish it.
```

Comment:
looking at the code a little bit, I'm not clear on why this would need to be a test case. I feel like if you always pass a span from one method to another, and the subsequent method calls span.finish(), it'll be handled and submitted back to the tracer finished spans.

Reply:
I don't remember the exact details, but back at the time when prototyping support for different frameworks in OT (mostly Tornado, which used to have it's own propagating mechanism) became tricky.

I'm up for re-reviewing this test case and drop it if we don't need it anymore, now that we use `Contextvars` mostly, that is (we also used to have more testbed samples, but we ended up removing them once they were not really representative).